README_dev.md

```diff
@@ -235,3 +235,5 @@ record AuthenticationStatus(
   - add expired flag to AuthenticationStatus
 - 0.6.0
   - add possibility to attach Metadata to Authentication-Requests
+- 0.8.0
```

Comment:
this is a breaking change and will be 1.0.0

Reply:
fixed